### PR TITLE
Get code coverage of several packages to 100%

### PR DESCRIPTION
**Pull request by geeknoid**

### CODE REVIEW

File: mixer/adapter/list/regexList.go [hunk 1]
```diff
@@ -55,10 +55,8 @@ func parseRegexList(buf []byte, overrides []string) (*regexList, error) {
 	}
 
 	for _, override := range overrides {
-		exp, err := regexp.Compile(override)
-		if err != nil {
-			return nil, err
-		}
+		// cannot fail, override syntax was checked in the Validate method
+		exp, _ := regexp.Compile(override)
```
Comment (ldemailly) [inline mixer/adapter/list/regexList.go]: is this really better code ? sometimes 100% coverage is worse than 99%
(like something isn't supposed to happen, until someone changes Validate and miss something and now it fails silently in production)

File: mixer/adapter/list/list.go [hunk 2]
```diff
@@ -31,6 +31,8 @@ import (
 	"sync"
 	"time"
 
+	"regexp"
```
[the remark this answers is not in the document]
Comment (geeknoid) [reply mixer/adapter/list/list.go]: I ran bin/fmt.sh and a ton of formatting changes were done across the source base. Apparently, once again the linter presubmit check is no longer working :-(

Anyway, I fixed these specific issues, I'll fix the overall formatting in a distinct PR.

File: mixer/adapter/list/list_test.go [hunk 3]
```diff
@@ -409,16 +433,70 @@ func TestRegexList(t *testing.T) {
...
+			"(", // invalid regex
+		},
+		EntryType: config.REGEX,
+	}
+	info := GetInfo()
+	b := info.NewBuilder().(*builder)
+	b.SetAdapterConfig(cfg)
+
+	err := b.Validate()
+	if err == nil {
```
[the remark this answers is not in the document]
Comment (geeknoid) [reply mixer/adapter/list/list_test.go]: Done.

File: mixer/adapter/list/regexList.go (hunk 1 above)
Comment (geeknoid) [reply mixer/adapter/list/regexList.go]: There are tests to ensure Validate is doing the right thing.

**Comment by geeknoid** [comment]
PTAL. All test failures appear to be flakes.

**Pull request merged**